integrations/hibob-service-user/syncs/unified-employees.ts

```
@@ -0,0 +1,42 @@
+import type { NangoSync, ProxyConfiguration } from '../../models';
+import { toStandardEmployee } from '../mappers/to-standard-employee';
```

ESLint

[error] Missing file extension for "../mappers/to-standard-employee"

(import/extensions)